packages/uniorg-parse/src/parser.ts

```diff
@@ -821,9 +846,13 @@ class Parser {
     this.r.resetOffset(contentsEnd);
     this.r.advance(this.r.line());
     this.parseEmptyLines();
-    const _end = this.r.offset();
 
-    return u('src-block', { affiliated, language, value });
+    return u('src-block', {
+      affiliated,
+      language,
+      value,
+      ...this.getPosition(contentsBegin, contentsEnd),
```

Comment:
This comment is still valid:
> ❗🐛 same here and all blocks below: begin/end should be wider

For all blocks, the begin/end should be wider than `contenstBegin`/`contentsEnd` and include `#+begin_xxx` and `#+end_xxx` lines.

```suggestion
    this.parseEmptyLines();
    const end = this.r.offset();

    return u('src-block', {
      affiliated,
      language,
      value,
      ...this.getPosition(begin, end),
```